wappsto/connection/communication.py

```diff
@@ -815,6 +857,15 @@ def receive(self, decoded):
                     self.remove_id_from_confirm_list(decoded_id)
 
                 elif decoded.get('result', False):
+                    value = decoded['result'].get('value', True)
+                    if value is not True:
+                        uuid = value['meta']['id']
+                        data = value['data']
+                        for device in self.instance.device_list:
```

Comment:
You should use the function 'get_by_id'

Reply:
the method created in branch: Support-sending-DELETE-requests, so to use it I need to wait for it to be merged